sprint_mission1/style.css

```
@@ -0,0 +1,197 @@
+* {
+	box-sizing: border-box;
+}
+
+:root {
+  --element-gray-color: #374151;
+  --element-sky-color: #3692FF;
+  --btn-hover-color: #1967D6;
+  --btn-active-color: #1251AA;  
+  --btn-visited-color: #9CA3AF;
+  --white-color: #FFFFFF;
+  --bg-sky-color: #CFE5FF;
+  --emelent-gray-color: #9CA3AF;
+  --bg-navy-color: #111827;
+}
+
+body {
+  margin: 0;
+  color: var(--element-gray-color);
+}
+h1 {
+  font-size: 3vw;
+}
+.element-click {
+  cursor: pointer;
+}
+
+.element-click:hover{
+  background-color: var(--btn-hover-color);
+}
+
+.element-click:active{
+  background-color: var(--btn-active-color);
+}
+
+.element-click:visited{
+  background-color: var(--btn-visited-color);
+}
+
+.btn-login {
+  background-color: var(--element-sky-color);
+  color: var(--white-color);
+  text-decoration: none;
+  border-radius: 5px;
+  width: 128px;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 0;
+  flex-shrink: 0;
+}
+
+.btn-go-looking {
+  background-color: var(--element-sky-color);
+  color: var(--white-color);
+  text-decoration: none;
+  border-radius: 99999px;
+  width: 100%;
+  height: 56px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.empty-center{
+  width: 1200px;
+}
+
+.logo-set {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 21px;
+}
+
+.header-top-line {
+  display: flex;
+  justify-content: center;
```

```suggestion
  justify-content: space-between;
```
`span` 태그를 이용해 빈공간을 만들기 보다는 css 속성을 이용해 배치를 해주시면 더 좋을 것 같습니다. 🤗